test/plugins/test/dragresize.js

```diff
@@ -0,0 +1,116 @@
+(function() {
+    'use strict';
+
+    var assert = chai.assert;
+
+    describe('imageScaleResize', function() {
+
+        var url = 'http://localhost/url_test';
+
+        describe('default imageScaleResize "both"', function() {
+            this.timeout(35000);
+
+            before(function(done) {
+                Utils.createCKEditor.call(this, done, {extraPlugins: 'ae_dragresize'});
+            });
+
+            after(Utils.destroyCKEditor);
+
+            beforeEach(Utils.beforeEach);
+
+            afterEach(Utils.afterEach);
+
+            it('The image, that is selected, has all resize handlers', function() {
```

Comment:
The combination of `describe` and `it` should make sense. Here is what the developer, who is reading the tests will get now:

`imageScaleResize default imageScaleResize "both" The image, that is selected, has all resize handlers`


Please look [here](https://mochajs.org/#getting-started) for an example how to use describe and it